### ISSUE TITLE
Having comments automatically printed on screen.

### ISSUE THREAD

**Issue by GabrieleBattaglia** [tage64/chess-cli#17]
Hi. it would be nice to have a customizable switcher in Chess-CLI that let's decide users to have or not the comments automatically printed on the screen while browsing game's move.
To me, while I'm watching a game by up/down arrows, it would be great to automatically read the comments if present when I reach its move.
You can provide a switcher for those of us that eventually do not like this behaviour.

**Comment by GabrieleBattaglia** [comment]
Hi Tage. Any chance to have this implementation?

**Comment by tage64** [comment]
Yes. I think it is a good idea. I will try to implement it when I get time.

**Comment by lucad1971** [comment]
I completely agree with this propossal... would be super useful

**Comment by GabrieleBattaglia** [comment]
Thanks. Any chance to have it implemented?--Sent from starship Gabryphone 12ProMax    Captain Battaglia.Il giorno 24 gen 2025, alle ore 14:41, lucad1971 ***@***.***> ha scritto:﻿
I completely agree with this propossal... would be super useful
